images/linux/scripts/installers/yq.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -e
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/install.sh
+
+# Test to see if the software in question is already installed, if not install it
+echo "Checking to see if the installer script has already been run"
+if [ -z $YQ_VERSION ]; then
+    YQ_VERSION=v4.9.6
```

Comment:
```suggestion
    YQ_VERSION=$(curl -s https://api.github.com/repos/mikefarah/yq/releases/latest | jq -r '.tag_name')
```

from https://github.com/Yann-J/virtual-environments/blob/203224ed3d218f7e0b14af71cdd0f772308412d4/images/linux/scripts/installers/nvm.sh#L9